src/flow/src/batching_mode/engine.rs

```diff
@@ -375,6 +376,8 @@ impl BatchingEngine {
         let res = task
             .gen_exec_once(&self.query_engine, &self.frontend_client)
             .await?;
+        // TODO(discord9): found a better way to make sure results get into result table instead of waiting
+        tokio::time::sleep(Duration::from_secs(1)).await;
```

Comment:
So the `gen_exec_once()` can't ensure this? Is it possible to use oneshot channels or something to get notifications?

Reply:
> So the `gen_exec_once()` can't ensure this? Is it possible to use oneshot channels or something to get notifications?

apprently there is no strict happen-before relation for two query in two seprarate connection, i.e.:
at time 0 conn 1: insert into table
at time 1 conn 2: select from that table, doesn't promise the `insert into` result must show up
and there is no way to sync anything, so can only sleep